Add a parameter for the cupertino button text style

daniel-possienke opened the pull request:
Because the text color of a `CupertinoDialogAction` can't be change by the theme in the stable flutter version, I've added a parameter to set the color separately.

larryaasen commented:
@daniel-possienke Thanks for this PR. I like this new feature and would like to merge this.

However, there are a few additional changes I would like. Can you add a line to the widget test `test UpgradeWidget Cupertino` using `cupertinoButtonTextStyle`, and add the `cupertinoButtonTextStyle` parameter to the Customization section (alphabetized) in the README file?

daniel-possienke commented:
@larryaasen Thanks for your feedback. I've added the `cupertinoButtonTextStyle` to the `test UpgradeWidget Cupertino` and I've added the new parameter to the README file.

larryaasen commented:
@daniel-possienke These latest changes look good so I will merge now. I will put out an update to the package soon. Thanks.
